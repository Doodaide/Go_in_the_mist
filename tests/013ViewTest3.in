who
place G9
view
place H10
view
place I9
view
place J8
view
place K9
view
place L10
view
term