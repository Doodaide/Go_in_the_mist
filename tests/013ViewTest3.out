B
E16,.................................................
G18,xxxxxxxxxxxxxx...................................
S16,....xxx....xxx....xxx....xxx....xxx....xxx....xxx
C3,x......x......x......x......x......x......xxxxxxx
P16,.................................................
A18,xxxxxxxxxxxxxxxxx....xxx....xxx....xxx....xxx....
